skills/morph-ppt/reference/morph-helpers.py

```
@@ -0,0 +1,392 @@
+    _run("officecli", "add", deck, "/", "--from", f"/slide[{from_slide}]")
+
+    print(f"{BLUE}Setting morph transition...{NC}")
+    _run("officecli", "set", deck, f"/slide[{to_slide}]", "--prop", "transition=morph")
+
+    print(f"{BLUE}Listing shapes for ghosting reference:{NC}")
+    rc, out, _ = _run("officecli", "get", deck, f"/slide[{to_slide}]", "--depth", "1")
+    print(out)
+
+    # Verify
+    print(f"{BLUE}Verifying transition...{NC}")
+    rc, out, _ = _run("officecli", "get", deck, f"/slide[{to_slide}]", "--json")
```

Ruff (0.15.12)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)